[set_files.py]
  expected: TIMEOUT
